### PR TITLE
Convert BlankScreen into the clearScreen method

### PR DESCRIPTION
**Pull request by roscale**
`BlankScreen` is not a drawing and doesn't have a position, so it's more appropriate for it to be a method.

### CODE REVIEW

**Comment by roscale** [comment]
This is a stacked PR. You should merge #59 before merging this one. I don't know if GitHub will be smart enough to retarget this PR to ardent2 when #59 will be merged. We'll find out together. Otherwise, I will retarget it manually.

**Comment by TheToto42** [comment]
@roscale It has been retargeted properly, but can you fix the merge conflicts ?

File: lib/src/bt_device.dart [hunk 1]
```diff
@@ -303,6 +303,23 @@ class GYWBtDevice with ChangeNotifier implements Comparable<GYWBtDevice> {
     await _sendBTCommand(command);
   }
 
+  Future<void> clearScreen({Color? color}) {
```
Comment (TheToto42) [inline lib/src/bt_device.dart:306]: You must describe the method for the dartdoc library generation
```suggestion
  /// Reset the content of the screen and its background color
  Future<void> clearScreen({Color? color}) {
```

**Review by TheToto42 — CHANGES_REQUESTED** [review]

File: lib/src/bt_device.dart (hunk 1 above)
Comment (roscale) [reply lib/src/bt_device.dart]: Can we add a CI action for that?

**Review by TheToto42 — APPROVED** [review]

**Pull request merged**